templates/Makefile

```diff
@@ -76,7 +76,7 @@ component_install: replace_images
 
 ansible_install: component_install
 	cp -r $(PACKAGE_ANSIBLE_DIR) $(INSTALLDIR)/
-	ln -srf $(INSTALLDIR)/install/components $(INSTALLDIR)/ansible/playbooks/openshift/components
+	gln -srf $(INSTALLDIR)/install/components $(INSTALLDIR)/ansible/playbooks/openshift/components
```

Comment:
Need an OS ifdef (or alternative command)

Reply:
Sorry - that local hack wasn't intended to be part of this PR.